### DISCUSSION TITLE
Welcome to Coloris Discussions!

### DISCUSSION THREAD

**Discussion by mdbassit** [mdbassit/Coloris#101] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Olganix** [comment]
Hi.
Thank for yours tools and thank to continue to upgrade it. 
I'm on modifying it to add the possibility to add name on color for swatches. I may be make a request on it feature.
It's because sometime 2 colors could be near, and a small area couldn't give the differences between them and user could make some mistake. 
Do you have thinking about the possiblity to add a button to create a new swatch to add in the list ? 
(and may be a option to save into localStorage ? )

As a share Ideas: 
instead of using " style=\"color: " + swatch + ";\"  "
considere using the ` or ', witch avoid using \" inside "", make it the code a better look.
I personnaly begin to prefer `` for Html because it's also deal with backspace (it's use the \n of ide, don't need to write it in the string)

Have a good day.

**Comment by Olganix** [comment]
My bad, I was modifying the dist js. you already clean that point of `` on src js. 

But another point, if you want to clean a little : 
You give the swatch value to the content/html of element, and you hide it by text_indent: -1000px and overflow: hidden on parent. 
After, on click, you take the text version of the html to fill the selector.
But you could put the in attribut (generaly named "data-XXX" ) and just use element.getAttribut("data-XXX") to get it back.

**Reply by mdbassit** [reply]
You suggestion doesn't add anything useful to the existing implementation.

I'm locking this announcement as it's not the right place to discuss new ideas. If you have any suggestions, please create a new discussion in the **Ideas** category.
